src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +71,55 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+
+class Block{
+	private String blockNumber;
+	
+	public Block(String blockNumber){
+		this.blockNumber = blockNumber;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.